packages/uikit/scss/_pagination.scss

```diff
@@ -35,7 +47,9 @@
 	  background-color: transparent;
 	  border-width: 0;
 
-    &:hover,
+    &:hover {
+      color: black;
+    }
     &:focus {
       background-color: $pagination-hover-bg;
     }
```

Comment:
I don't think we want a different behavior for focus and hover. If we're not doing background color for the unstyled hover we should probably also lose it for focus and make the behaviors match. focus will have a border anyway.

@jordan-a-young thoughts?

Reply:
Awesome. I believe the focus outline is already inherited, so just get rid of the bg color and make hover & focus color=black.